rocksdb/columnfamily.nim

```diff
@@ -19,43 +19,48 @@
 
 {.push raises: [].}
 
-import
-  ./rocksdb
+import ./columnfamily/cfhandle, ./rocksdb
 
 export rocksdb
 
 type
   ColFamilyReadOnly* = object
     db: RocksDbReadOnlyRef
-    name: string
+    cfHandle: ColFamilyHandleRef
 
   ColFamilyReadWrite* = object
     db: RocksDbReadWriteRef
-    name: string
+    cfHandle: ColFamilyHandleRef
 
 proc withColFamily*(
-    db: RocksDbReadOnlyRef,
-    name: string): RocksDBResult[ColFamilyReadOnly] =
+    db: RocksDbReadOnlyRef, name: string
```

Comment:
I think this might be a good opportunity to rename this function - `withXxx` is typically used for templates that scope access to something via a body, ie `withColFamily(name): usefamilyinthisscopeonly`

Reply:
Sure, I'll rename these functions.